Packs/DomainTools_Iris/Scripts/DomainToolsCheckTags/check_tags.py

```diff
@@ -0,0 +1,41 @@
+from CommonServerPython import *
+
+from typing import Dict, Any
+import traceback
+
+
+def check_tags(args: Dict[str, Any]) -> CommandResults:
+    incident_id = args['incident_id']
+    domain_tags = args['domain_tags']
+    domain_tags_set = set([domain_tag['label'] for domain_tag in domain_tags])
+
+    malicious_tags = args['malicious_tags']
+    malicious_tags_set = set(json.loads(malicious_tags))
```

Comment:
Please set a const variable at the top of this file instead of an external file.
Something like:
```python
MALICIOUS_TAGS = ["xyz", "footyphishing","Demistodemo","test_tag", "blocklist"]
```

Reply:
this is supposed to be a dynamic vars which users can freely change. Will be moving this as an argument rather getting the values from an external file. Thank you!